lib/jenkins.js

```diff
@@ -68,7 +79,7 @@ module.exports = function (args) {
         command.push(getBin('mocha'));
     }
 
-    command.push('--reporter ' + require.resolve('spec-xunit-file'));
+    command.push('--reporter ' + escape(require.resolve('spec-xunit-file')));
     command.push(args.join(' '));
```

Comment:
Actually, this might need to be escaped.  Basically wrap `args` with `shellEscape`